nuxt-frontend/components/backend-development/PartnerWithUsSection.vue

```diff
@@ -0,0 +1,189 @@
+              :key="item"
+              class="flex flex-col mb-10 item opacity-0"
+              :class="[isVisible ? `item-${index} !opacity-100` : 'hidden']"
+            >
+              <div class="flex gap-[1.69rem]">
+                <img
+                  :src="item.image"
+                  class="w-9 h-9 xl:w-14 xl:h-14 object-contain"
+                  :alt="item.title"
+                />
+                <div class="flex flex-col gap-4">
+                  <span
+                    class="text-xl leading-8 font-semibold xl:sub-h1-semibold"
+                    >{{ item.title }}</span
+                  >
+                  <span
+                    class="text-[1rem] leading-[1.688rem] xl:sub-h3-regular text-white/[0.8]"
+                    v-html="item.description"
+                  ></span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- ---------------------DESKTOP UI END-------------------- -->
+    </div>
+  </section>
+</template>
+<script setup>
+import maintainance from "@/assets/images/backend-development/partnerwithus/maintanance.webp";
+import moneyback from "@/assets/images/backend-development/partnerwithus/moneyback.webp";
+import user from "@/assets/images/backend-development/partnerwithus/user.webp";
+const { $mixpanel } = useNuxtApp();
+const isAnimated = ref(false);
+const isVisible = ref(false);
+const items = [
+  {
+    title: "Skin In The Game",
+    image: maintainance,
+    description: `To prove our skin in the game, we offer <span class="text-[#FF9472] text-4 leading-[1.688rem] font-semibold xl:sub-h3-semibold">6 Months of Free Post Deployment Maintenance</span> for all the work we do on your project. If any bugs arise in the project within 6 months of deployment, we'll do it for free. This approach ensures careful development, as neglect could result in significant expenses.`,
+  },
+  {
+    title: "100% Risk Free Experience / Money Back Guarantee",
+    image: moneyback,
+    description: `To walk the talk, we offer a <span class="text-[#FF9472] text-4 leading-[1.688rem] font-semibold xl:sub-h3-semibold">100% Money Back Guarantee for two weeks of work at any time during our engagement</span>. If you work with us, you will get a feedback survey every Monday. If you ain't happy with the work we did in the past week, get your money back.`,
+  },
+  {
+    title: "Achieving Excellence - One step at a time",
+    image: user,
+    description: `Whether your goal is user growth, user engagement, revenue growth, user satisfaction, improved user experience, unforgettable branding, customer loyalty, or all of them, we know how to achieve it, one step at a time!`,
+  },
+];
+
+onMounted(() => {
+  window.addEventListener("scroll", handleScroll);
+});
+
+onUnmounted(() => {
+  window.removeEventListener("scroll", handleScroll);
+});
+function handleScroll() {
+  const stickyparent = document.querySelector(".stickyparent");
```

Comment:
use content related class name, cause it will select the very first class with name `stickyparent` in document